modules/sdk-lib-mpc/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "./dist/src/index.d.ts",
   "scripts": {
     "test": "yarn unit-test",
-    "unit-test": "nyc -- mocha",
+    "unit-test": "nyc -- mocha --recursive test",
```

Comment:
Could add context on to why we need these flags + the etension change in the .mocharc?

Reply:
`spec: ['test/unit/*.ts']` didn't run tests under sub-directories of `unit/`.

it's after the same change from #3807

Reply:
I think you can just change the spec to be. Is there a reason to have `.js` files here?
```
['test/unit/**/*.ts']
```

Reply:
WIth this the `--recursive` flag might be redundant?

Reply:
I don't disagree with `['test/unit/**/*.ts']`, but with #3807 already in `master`, it's better to be consistent IMO.